app/controllers/nonsipp/landorpropertydisposal/LandOrPropertyDisposalListController.scala

```diff
@@ -212,6 +212,13 @@ class LandOrPropertyDisposalListController @Inject()(
         numberOfDisposals >= maxPossibleNumberOfDisposals ||
         allPropertiesFullyDisposed
 
+      statusMap = for {
+        lpIndex -> disposals <- request.userAnswers.map(LandOrPropertyDisposalProgress.all(srn))
+        disposalIndex -> status <- disposals
+      } yield (lpIndex, disposalIndex) -> status
+
+      inProgressUrl = statusMap.collectFirst { case (_, SectionJourneyStatus.InProgress(url)) => url }
```

Comment:
Let's discuss this logic, when the other priority tickets are done please, as it can be solved in multiple ways

Reply:
sure